spec/unit/ace/plugin_cache_spec.rb

```diff
@@ -13,6 +12,7 @@
   let(:params) { '?checksum_type=md5&environment=production&ignore=.hg&links=follow&recurse=true&source_permissions=' }
 
   before do
+    WebMock.disable_net_connect!
```

Comment:
please move this to an `after` block on the acceptance tests.

I'd rather have *everything* running under `disable_net_connect!` and have the acceptance tests be responsible for cleaning up than having to remember on the next test that might trigger a webrequest to disable it there, too.